docs/index.md

```diff
@@ -1,15 +1,18 @@
 ---
-title: AI Validation
-summary: Documenting the processes of the AI Validation Team at the Ministry of the Interior and Kingdom Relations (Min BZK) in The Netherlands.
+title: Home
+summary: Documenting the processes and work of the AI Validation Team at the Ministry of the Interior and Kingdom
+    Relations (Min BZK) in The Netherlands.
 ---
```

Comment:
In the `mkdocs.yml` we have a site_description, so we could remove it here.

And maybe a personal preference, but I prefer not to have a "Home" page. What do you think of using `Welcome!` instead?

```suggestion
# Welcome!
```

Reply:
Took most of this on board (but kept "Home")